src/components/Hello.tsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import { Button, StyleSheet, Text, View } from "react-native";
+
+export interface Props {
+  name: string;
+  enthusiasmLevel?: number;
+}
+
+interface State {
+  enthusiasmLevel: number;
+}
+
+export class Hello extends React.Component<Props, State> {
```

Comment:
Not sure this belongs in a RN update PR? I'd prefer UI changes etc in a separate PR